File: nan21.dnet.module.hr/nan21.dnet.module.hr.ui.extjs/src/main/webapp/src/ds/EmployeeContactRelationshipLovDs.js
 
   	 
Ext.define("net.nan21.dnet.module.hr.employee.ds.model.EmployeeContactRelationshipLovDs" , {
	extend: 'Ext.data.Model',
	fields: [  		
		{name:"id",type:"int", useNull:true }, 		
		{name:"clientId",type:"int", useNull:true }, 		
		{name:"name",type:"string" }, 		
		{name:"active",type:"boolean" }
	]
});
	
Ext.define("net.nan21.dnet.module.hr.employee.ds.param.EmployeeContactRelationshipLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 

File: nan21.dnet.module.hr/nan21.dnet.module.hr.presenter/src/main/java/net/nan21/dnet/module/hr/employee/ds/model/EmployeeContactRelationshipLovDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms. 
 */
package net.nan21.dnet.module.hr.employee.ds.model;

import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeLov;
import net.nan21.dnet.module.hr.employee.domain.entity.EmployeeContactRelationship;

@Ds(entity = EmployeeContactRelationship.class, jpqlWhere = " e.active = true ", sort = { @SortField(field = EmployeeContactRelationshipLovDs.fNAME) })
public class EmployeeContactRelationshipLovDs extends
        AbstractTypeLov<EmployeeContactRelationship> {

    public EmployeeContactRelationshipLovDs() {
        super();
    }

    public EmployeeContactRelationshipLovDs(EmployeeContactRelationship e) {
        super(e);
    }

}
